sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/SampleCheckpointStore.java

```diff
@@ -81,9 +84,11 @@ private String prefixBuilder(String fullyQualifiedNamespace, String eventHubName
                     || partitionOwnershipMap.get(partitionOwnership.getPartitionId()).getETag()
                     .equals(partitionOwnership.getETag());
             })
-            .doOnNext(partitionOwnership -> logger
-                .info("Ownership of partition {} claimed by {}", partitionOwnership.getPartitionId(),
-                    partitionOwnership.getOwnerId()))
+            .doOnNext(partitionOwnership ->
+                logger.atInfo()
+                    .addKeyValue(PARTITION_ID_KEY, partitionOwnership.getPartitionId())
+                    .addKeyValue("ownerId", partitionOwnership.getOwnerId())
```

Comment:
Is there a reason we don't use the constant here?

Reply:
great catch!